### PR TITLE
Add feedback to beman overview presentation.

### PR DESCRIPTION
**Pull request by camio**

### CODE REVIEW

File: presentations/beman_overview_wg21_202406.org [hunk 1]
```diff
@@ -175,9 +201,20 @@ int main() {
...
 #+BEGIN_QUOTE
 It's the End of the World as We Know It (And I Feel Fine)
 
     - R.E.M. 1987
 #+END_QUOTE
+
+# DJS: I suggest incorporating a live demo:
+# 1. discourse.bemanproject.org
+# 2. The Beman.Optional26 repository. Also the Godbolt
+# 3. The Beman standard to show what's there
```
Comment (camio) [inline presentations/beman_overview_wg21_202406.org:220]: Another thing I think should be incorporated in the presentation is probably a slide and discussion of each on the [four core principles](https://github.com/beman-project/beman/blob/c73fb3f48845b3677fce7d2152487c71f67ee033/docs/beman-standard.md?plain=1#L15-L29) of the Beman standard.

File: presentations/beman_overview_wg21_202406.org [hunk 2]
```diff
@@ -47,16 +47,33 @@ Founded at C++Now in 2024 the project strives to aggregate libraries proposed fo
...
+
+# DJS: Again, I think we shouldn't say anything that can be interpreted
+# anti-Boost here. The Boost project is still introducing quality libraries into
+# the wider ecosystem. However, Github has increasingly become the ubiquitous
+# "repository of free C++ class libraries".
+#
+# On second thought, I think we should just strike this last bullet. The key
+# observation is that quality libraries are coming from elsewhere, which is
+# a good thing, but it makes it much more difficult to identify and use
+# libraries intended for standardization.
```
Comment (JeffGarland) [inline presentations/beman_overview_wg21_202406.org:75]: Agree - changed to:

- By ~2018-202x most libraries from elsewhere

File: presentations/beman_overview_wg21_202406.org [hunk 3]
```diff
@@ -47,16 +47,33 @@ Founded at C++Now in 2024 the project strives to aggregate libraries proposed fo
...
-The short version: the world that Boost was born in no longer exists. In addition, Boost is no longer the source of the majority of library proposals. WG21 and users have lost something big - a single place to go see c++ next libraries.  Boost cannot be molded into what we need for a variety of reasons. Beman is a new effort (sponsored by the Boost Foundation) to provide that one place to go - using the latest tooling and best practices. Boost libraries are welcome in Beman as are any proposed for standardization.
+The short version: the world that Boost was born in no longer exists. In addition, Boost is no longer the source of the majority of library proposals. WG21 and users have lost something big - a single place to go see C++ next libraries.  Boost cannot be molded into what we need for a variety of reasons. Beman is a new effort (sponsored by the Boost Foundation) to provide that one place to go - using the latest tooling and best practices. Boost libraries are welcome in Beman as are any proposed for standardization.
 
 reviews are stuck in 1990 - should use tools!
 
 #+END_NOTES
+
+# DJS: "reviews are stuck in 1990 - should use tools!" could be interpreted as a
```
Comment (JeffGarland) [inline presentations/beman_overview_wg21_202406.org:56]: removed -- reminder these are speakers notes not on screen.  Normally when I distribute the pdf of these they aren't in the content -- that said, it's good to have the right notes to stay on point.

File: presentations/beman_overview_wg21_202406.org [hunk 4]
```diff
@@ -86,7 +103,7 @@ reviews are stuck in 1990 - should use tools!
 #+BEGIN_NOTES
 - A distribution of libraries proposed for standardization
 - Including non-Boost reviewed libraries
-- Separate from from Boost
+- Separate from Boost
```
Comment (JeffGarland) [inline presentations/beman_overview_wg21_202406.org:106]: removed the boost references here -- not needed

File: presentations/beman_overview_wg21_202406.org [hunk 5]
```diff
@@ -113,6 +134,8 @@ int main() {
         std::cout << "\"for each loop\" over C++26 optional: opt = " << i << "\n";
     }
 #+END_SRC
+# DJS: It might be good to mention that this is available on Godbolt and our
+# intent is for Godbolt to track Beman libraries as well.
```
Comment (JeffGarland) [inline presentations/beman_overview_wg21_202406.org:138]: have live demo for Optional26

File: presentations/beman_overview_wg21_202406.org [hunk 6]
```diff
@@ -134,9 +157,11 @@ int main() {
   - authors don't have to use Beman repos
 ** [[https://github.com/beman-project/beman/blob/main/docs/FAQ.md][FAQ]]
 - Do I need a paper before joining? - No
+# DJS: We should mention that we need a paper number before you can add a
+# library into Beman.
 - Library not for the C++ standard? Can I add to Beman Project? - No
 - Personal/work project no interest in becoming a standard library. Does the Beman Project have anything for me? - Yes
-- Licensing: Boost, MIT, Apache with LLVM exception
+- Licensing: Apache with LLVM exception, Boost, MIT
```
Comment (JeffGarland) [inline presentations/beman_overview_wg21_202406.org:164]: done

File: presentations/beman_overview_wg21_202406.org [hunk 7]
```diff
@@ -175,9 +201,20 @@ int main() {
...
+# Another thing you could do is ask folks to look at the Beman standard.
 ** End
 #+BEGIN_QUOTE
 It's the End of the World as We Know It (And I Feel Fine)
 
     - R.E.M. 1987
 #+END_QUOTE
+
+# DJS: I suggest incorporating a live demo:
+# 1. discourse.bemanproject.org
```
Comment (JeffGarland) [inline presentations/beman_overview_wg21_202406.org:218]: done

**Comment by camio** [comment]
No longer needed.

**Pull request closed without merging**